src/esssans/isis/visualization.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2024 Scipp contributors (https://github.com/scipp)
+"""
+Plotting functions for ISIS data.
+"""
+from typing import Any
+
+import scipp as sc
+
+
+def plot_instrument(da: sc.DataArray, pixels_per_tube: int = 512, **kwargs: Any) -> Any:
+    """
+    Plot a 2-D instrument view of a data array.
+
+    This is an alternative to the `scn.instrument_view` function, avoiding the 3-D
+    rendering of the instrument. The exact X and Y coordinates of the pixels are
+    used for the 2-D plot.
+
+    Parameters
+    ----------
+    da:
+        The data array to plot. Must have a 'position' coord and a single dimension.
```

Comment:
Is it not possible to flatten if there are more than one dimension?

Reply:
Sure, but this is for ISIS data, which has been loaded with Mantid, so why complicate things? It can be extended once/if required?

Reply:
I had considered doing this, but then it would complicate since we would need to maybe support summing some dims but flattening others, I didn't think it was worth it at this point. After all, this is not an ESS instrument.